demos/workflows/mcp-gh-tools.ai/workflow.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: maestro/v1alpha1
+kind: Workflow
+metadata:
+  name: openai_test_mcp_ollama
+  labels:
+    app: testapp_mcp_ollama
+spec:
+  template:
+    metadata:
+      labels:
+        app: testapp_mcp_ollama
+    agents:
+      - test
+    prompt: |
+      I want to learn about the authenticated GitHub user (myself).
+
+      1. Use the get_me() tool in Github MCP to call to get my user profile, and then output a summary of my profile.  
```

Comment:
I think it's good to get this checked in as an example, no need to change.. but:

In this case you will be sending the combination of the workflow prompt and agent prompt together at least for a one-step workflow, which has duplication.